src/sass/mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin mobile {
+  @media screen and (max-width: 767px) {
+    @content;
+  }
+}
```

Comment:
Love this, makes things much cleaner!

Reply:
In more extensive projects, I usually go for something like this:

```scss
// variables.scss

$breakpoints: (
  "xs": (0px 768px),
  "sm": (768px 992px),
  "md": (992px 1200px),
  "lg": (1200px nil)
);
```

```scss
// functions.scss

@function get-breakpoint($key) {
  @if map-has-key($breakpoints, $key) {
    @return map-get($breakpoints, $key);
  } @else {
    @error "Breakpoint \"#{$name}\" does not exist.";
  }
}

@function get-breakpoint-query($key, $isMaxWidth) {
  @if $isMaxWidth {
    @return "(max-width: #{nth($key, 2) - 1px})";
  } @else {
    @return "(min-width: #{nth($key, 1)})";
  }
}
```

```scss
// mixins.scss

@mixin breakpoint($bp, $isMaxWidth: false, $isScreenOnly: true) {
  $breakpoint: get-breakpoint($bp);
  $query: get-breakpoint-query($breakpoint, $isMaxWidth);

  @if $isScreenOnly {
    $query: "only screen and #{$query}";
  }

  @media #{$query} {
    @content;
  }
}
```

```scss
// my-styles.scss

@include breakpoint('sm') {
  // styles...
}
```

Suuuuper extensible

Reply:
Ah nice! Never used conditions in SASS before, but this is a great use case for them!